grakn-graql/src/main/antlr4/ai/grakn/graql/internal/antlr/Graql.g4

```diff
@@ -30,7 +30,7 @@ mean           : MEAN     'of' ofList      ('in' inList)? ';' ;
 std            : STD      'of' ofList      ('in' inList)? ';' ;
 sum            : SUM      'of' ofList      ('in' inList)? ';' ;
 degrees        : DEGREES ('of' ofList)?    ('in' inList)? ';' ;
-cluster        : CLUSTER                   ('in' inList)? ';' clusterParam* ;
+cluster        : CLUSTER ('of' id    )?    ('in' inList)? ';' clusterParam* ;
```

Comment:
would this work: compute cluster in xxx of v123;

Reply:
No, has to be the other way around. But this is true for all the methods so far. We can change it later.